web/src/main/java/org/cbioportal/web/config/CustomObjectMapper.java

```diff
@@ -28,6 +26,5 @@ public CustomObjectMapper() {
         mixinMap.put(Sample.class, SampleMixin.class);
         mixinMap.put(TypeOfCancer.class, TypeOfCancerMixin.class);
         super.setMixInAnnotations(mixinMap);
-        super.configure(JsonGenerator.Feature.WRITE_NUMBERS_AS_STRINGS, true);
```

Comment:
we must test the built warfile and make sure all visualizations work after making this change.